docs/modules/ROOT/pages/install-clc.adoc

```diff
@@ -30,6 +36,12 @@ ZIP (Apple Silicon)::
 +
 . Go to the https://github.com/hazelcast/hazelcast-commandline-client/releases/latest[latest release page], and locate the ARM compressed file (`hazelcast-clc_v{full-version}_darwin_arm64.zip`).
 . Download and unzip the file.
+. Remove the `clc` binary from quarantine if you get a security warning when running it:
++
+[source,shell,subs="attributes"]
+----
+xattr -d com.apple.quarantine hazelcast-clc_v{full-version}_darwin_amd64/clc
```

Comment:
I think this should be arm, right?

Reply:
Thanks, updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/245/commits/8aa04515f273aeca9661a01a90277b98ba1fcc0f